### DISCUSSION TITLE
software pirata

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#681] [Discussão]
O que vocês como devs, acham dessa pratica não só de devs ou usuários comuns, mas de empresas grandes também?

**Comment by expalmer** [comment]
Independente de lei, de pessoas, de qualquer coisa....pra mim, software pirata é errado. Se alguém fez algo e quer cobrar, e tem outro alguém que quer usar...então pague por isso. Eu não baixo nem música, filmes e séries, até isso acho errado.

**Comment by AmandaFonseca** [comment]
Sobre as empresas grandes até a Globo usa : http://www.alagoas24horas.com.br/1055424/ops-jornal-da-globo-usa-windows-pirata-ao-vivo/.
Porém eu acho que é uma hipocrisia reclamar da Política consumir softwares piratas, moro de aluguel periferia e ainda não ganho bem, mesmo assim fiz um esforço em comprei a ultima versão do Office.

Sendo bem sincera as vezes eu  não me controlo e baixo um torrent ou música mas evito ao máximo, faço isso somente quando não tenho condições meesmo ou estou muito curiosa para assistir um filme ou série. Eu comecei a ter esse pensamento depois que entrei na área  é claro mas principalmente depois que comecei a estudar fluxo de energia  porque segundo as leis energéticas toda vez que vc tenta tirar vantagem de alguém e se culpa por isso só gera mais desgraça na sua vida, fora que isso alimenta a crença de que você não confia em si mesmo para pagar as suas coisas, gerando um bloqueio na vida maaaaasss isso é uma crença de cunho pessoal/filosófico que foge do tema dessa Issue.

**Comment by calaca** [comment]
Eu diria que o problema de querer comprar e não ter dinheiro, apontado pela @AmandaFonseca , é o maior vilão nesse caso. Isso tratando-se das pessoas que tem um tiquinho de ética, rs. Às vezes realmente não temos como assinar aquele pacote de TV caro, nem aguentamos esperar um ano para ver algo que saiu hoje. Uma solução pra consciência pesada é comprar o que vc consumiu de maneira ilegal quando tiver dinheiro. :)

**Comment by mateusmiguel** [comment]
Já tive um monte de softwares piratas e um hd cheio de discografia. Com o tempo passei a 'me corrigir' e usar cada vez menos coisas piratas. Uso o Office online desde a última vez que formatei minha máquina, por exemplo. Não tenho mais instalado nela. Muita coisa é contornável hoje em dia. Tem versão free/alternativa/algo que te atenda de bastante coisa.

O únicos programas que (ainda) uso pirata na minha máquina hoje são o PS e uns plugins vst.

**Comment by mateusmiguel** [comment]
@calaca tbm faço isso sempre que posso!

**Comment by bernardocs** [comment]
Esse é um assunto, na minha concepção muito, **muito** simples, mas que dá trigger em pessoas que curtem se classificar moralmente como "white hat". Por ter conversado sobre isso com um amigo meu recentemente, vou dar minha opinião _sincera_ mesmo.

Essa discussão como um todo, ao meu ver, beira o _irrelevante_ .
Ora... somos profissionais de TI em pleno 2017. Nós, por definição, **dominamos o computador** como ferramenta. Sendo isso, em suma, a idéia de que você tem a liberdade **e a habilidade** de fazer **o que quiser** no seu computador e que não há nada que deveria impedi-lo de exercer tal liberdade.

Usei softwares piratas, em geral (SO's, jogos, IDE's, etc) durante muito tempo por simples fatos: 
- eu tinha a **necessidade** de ter alguns deles instalados; 
- antes de me estabilizar como desenvolvedor, eu era **pobre** (nem troco da padaria sobrava pra mim);
- eu tenho o pleno conhecimento para instala-los;

Eu acho _ridicularmente_ fácil falar sobre pirataria e moralidade quando você olha para um jogo na Steam a R$50, ou uma licença de US$70 do Sublime e questiona a moralidade acima da questão financeira e diz "Vou contribuir pq acho moralmente correto.. blablabla" e paga... Quando que, na verdade, tem muita gente que não consegue se dar ao luxo de fazer isso com tanta facilidade quanto você.

Importantíssimo denotar que eu não uso software pirata à anos, desde que descobri linux e desde que comecei a trabalhar como desenvolvedor. O que implica em duas coisa: 
- uso Linux para fins profissionais... e ele é gratuito.
- eu, agora, tenho dinheiro para poder comprar meus jogos da Steam e/ou uma licença do Windows (para jogar - PC Master Race né).

No mais, também acho importante denotar que existe limites pra esse tipo de pensamento goody-goody... você é um dev em 2k e fucking 17. Você **domina** o PC e é livre pra exercer esta liberdade com o que você **quiser**, desde que não prejudique as pessoas de exercerem a liberdade delas.

É o que eu sinceramente acho... :)

Cheers

**Comment by expalmer** [comment]
[treta] não se ofendam. Mas se você quer fazer algo e não pode por algum motivo, então não faça! Faça outra coisa, vai ler um livro...vai jogar bola na praça, vai visitar sua família. Se eu de alguma forma quiser fazer/ter algo que não posso, e faço qualquer coisa para fazer/ter...então os fins justificam os meios!?!

Faltou mais...

Para resumir, para mim é errado e ponto.(Mas para mim! Não para você, e eu não julgo ninguém por isso).
Sua cabeça é seu guia, faça aquilo que acredita ser o certo!

**Comment by weltonsanches** [comment]
Eu acho que como devs precisamos dessa ética de não piratear, se o software é pago junta uma grana pra comprar ou **busca uma alternativa open-source <3**. Somos nós quem desenvolvemos, é bem incoerente piratear algo.

Quanto a músicas e filmes, acho que os serviços de streaming tendem a ajudar diminuir a pirataria.

**Comment by kvnol** [comment]
O termo **software pirata** já é errado, não tem muita discussão sobre isso, mas temos o problema dos softwares custaram um absurdo o que faz muita gente (estou no pacote) não licenciar seu pacote Adobe.

Eu ouvi a seguinte frase do [Caio "Brogui" Novaes](https://twitter.com/brogui) em que ele dizia, quando mais novo:

> Quando eu puder pagar pelo produto, não vou mais piratear

E tirei isso para minha vida. Sempre que vou ganhando uma grana a mais e vejo que posso licenciar algum software que eu uso, vou lá e compro/assino.

_Update:_
Sou músico e música é algo que já pirateei e, sinceramente, não consigo mais. Uso Spotify para tudo, nem YouTube eu uso para escutar música (até por costume, já).

**Comment by andresantos-tech** [comment]
Acho que todo mundo aqui deve ser contra a pirataria, por questões éticas e etc. Tomando isso como verdade, vou tentar usar um outro enfoque. Vejam essas três questões:

- Para a Microsoft, é bom ou ruim que tantos usuários domésticos usem windows pirata?
- É bom ou ruim para uma produtora de grande porte (vamos colocar, sei lá, a HBO) que um episódio de uma de suas séries vaze perto do season finale?
- Se nunca tivesse havido pirataria de músicas (através de torrents e etc), hoje nós teríamos serviços como o Spotify?

Indo nessa linha, pirataria tem a mais a ver com "crime" ou com "demanda que o mercado ainda não conseguiu atender"? Eu por exemplo nem lembro quando foi a última vez que baixei uma música, mas desconfio que foi +ou- na mesma época em que comecei a usar o finado Rdio (e depois migrei para o Spotify).

**Comment by kvnol** [comment]
> a última vez que baixei uma música, mas desconfio que foi +ou- na mesma época em que comecei a usar o finado Rdio (e depois migrei para o Spotify)

A minha última vez foi nessa época também, algo entre 2013/14.

**Comment by rubenmarcus** [comment]
poxa o Spotify, não me atende por exemplo, gosto de Aphex Twin, e lá não tem nem metade dos albuns dele.Tem produções dele que ele soltou no soundcloud e removeu que você só encontra no soulseek e em alguns canais do youtube que em breve são removidos.

Assim como vejo vagas de front jr. pedindo conhecimento em pacote Adobe, mas o pacote Adobe completo é meio proibitivo pra um front jr.

**Comment by rubenmarcus** [comment]
Mas o enfoque do tópico, é mais por curiosidade, mas como a @AmandaFonseca  falou, ao meu ver a pirataria de um software por uma empresa, que pirateia sei lá pacote Adobe ou outras ferramentas pra 100 maquinas, isso só pra baratear sua produção, o que não afetaria tanto no seu lucro, bem pior, do que um usuário, que não pode pagar a mensalidade da Adobe Creative Cloud.por exemplo.

Vejo uma galera que argumenta "Po se voce nao consegue pagar, voce nem deveria estar no mercado". É não é bem assim né...

**Comment by kvnol** [comment]
> Vejo uma galera que argumenta "Po se voce nao consegue pagar, voce nem deveria estar no mercado". É não é bem assim né...

Esse tipo de galera é o mesmo que, quando tu pergunta "me indica uma máquina legal para programar e tal", te indica o Macbook Pro 17".

**Comment by kvnol** [comment]
> Os políticos fazem campanhas antipirataria mas colocam barreiras nos serviços que podem ajudar no combate da pirataria um exemplo tipo a Netflix existe um processo no senado tornar o serviço tão caro quanto os das operadoras[...]

É o problema do estado grande, né? Mas nem vou puxar esse assunto aqui (tem uma galera que ama), hahahaha.

**Comment by AmandaFonseca** [comment]
Acabei excluindo o meu comentário @kvnol  pq já se estressei muito essa semana por causa de discussões de política.

A galera as vezes leva para o outro lado e eu não estou afim...

**Comment by phhoffmann** [comment]
Po, tava pensando nisso ontem. Por necessidade profissional, preciso de várias coisas do pacote Adobe e R$175 é muita, muita coisa pra eu pagar mensalmente. Não trabalho sozinho, não sou o chefe.. preciso usar eles. O pacote custa R$175 reais, saca, não é qualquer trocado, dou o jeito que consigo dar e.. piratebay neles.

No meu pc, não tenho os programas, não baixo música, etc.

Talvez caia sim na questão da "pequena corrupção do dia a dia". Mas é difícil falar isso quando se tem tão pouco incentivo na área.

Sobre achar certo ou errado, acho bem simples na real. É roubo, né. haha, você quer ter uma coisa que custa um dinheiro X, você não pagou pra ter aquilo.. é roubo.

Enquanto isso, vou na mesma da frase que disseram ali em cima:

> Quando eu puder pagar pelo produto, não vou mais piratear

**Comment by caioincau** [comment]
Se é ferramenta de trabalho, deveria ser fornecido pelo seu empregador.

**Comment by rubenmarcus** [comment]
@icaioincau  isso cai também em conhecimento tecnico. Se o contratante te contratou sabendo conhecimento XY e foi pedido isso na vaga e após meses ele pede conhecimento XYZWO e não ajudou voce com cursos ou etc etc, ele espera q vc banque por fora. E ai?? é um paradoxo não é?

**Comment by caioincau** [comment]
Não, eu pediria tempo para estudar o conhecimento "faltante", como sempre fiz.

Você está fazendo dinheiro com a ferramenta alheia? E ainda sim não está pagando ela? Tá errado.

**Comment by RatoX** [comment]
> Acho que todo mundo aqui deve ser contra a pirataria, por questões éticas e etc. Tomando isso como verdade, vou tentar usar um outro enfoque. Vejam essas três questões:

Nem todo mundo cara, eu sou bem a favor e por questões éticas também. 

Não vou explicar o do por que mas sempre aconselho a lerem sobre a história do protocolo Samba a gente tá praticamente sentado em cima de pirataria.

**Comment by willianjusten** [comment]
Vi uma imagem interessante referente a isso hoje...

![screen shot 2017-08-31 at 12 20 58](https://user-images.githubusercontent.com/3991845/29933399-d45ee856-8e4d-11e7-9c4e-9237961b7c4c.png)

**Comment by ricardogouveia3** [comment]
> Esse tipo de galera é o mesmo que, quando tu pergunta "me indica uma máquina legal para programar e tal", te indica o Macbook Pro 17".

@kvnol Mas o macbook é uma ótima máquina pra programar...

**Comment by ricardogouveia3** [comment]
Vou trazer a questão da Apple

Todo mundo sabe que os computadores da Apple são caros; principalmente no Brasil, mas ainda assim no exterior eles estão acima da média no preço.
Sendo assim, o meu cálculo não vai bater no contexto nacional, mas vale a pena refletir:

Comprando um mac, você:
1. Não irá pagar o sistema operacional
2. Não irá pagar o pacote Office (Pages, Numbers e Keynote gratuitos)
3. Não irá pagar anti-virus (na maioria dos casos, nem vai ter)

Isso não exclui você de custos de softwares de uso específico (sublime, por exemplo; se bem que editor de texto é o que mais tem alternativa hoje em dia). Mas já corta algum custo.

Sou fã boy, então é tendencioso, mas acho que é válido.

**Comment by rubenmarcus** [comment]
ué se não for pra pagar software usa linux logo :P

**Comment by Rodrigo54** [comment]
Vamos analisar a situação hipotética da estoria de Joãozinho...

> Joãozinho é um grande desenvolvedor que trabalhou durante muito tempo num software de streaming de vídeo de um grande player do mercado. Hoje Joãozinho trabalha numa empresa concorrente a seus antigos empregadores, nesse novo emprego ele fara um software de streaming de vídeo.

_Obs: O código fonte do software de streaming de vídeo e todas as tecnólogas envolvidas são protegidas por leis de direitos autorais e são de propriedade intelectual privada da empresa._

**O que Joãozinho deve fazer?**

&nbsp; :heart:  &nbsp;Começar o novo projeto do zero.
:tada: Aproveitar do projeto anterior apenas o código feito por ele.
:smile: Mostrar para seu gerente as tecnologias usadas na sua antiga empresa sem mostrar códigos _(mostrando onde e não como foi feito)_.
:confused: Usar partes do código do projeto anterior como se fossem de licença GLP.
 :-1: Se o gerente permitir, forkar o projeto inteiro e começar a partir disso.
:+1: Reescrever o mesmo tipo de código de cabeça, sem duplicar exatamente as linhas de código do projeto anterior.

![saw](http://geradormemes.com/media/created/0ok8lz.jpg)

**Comment by calaca** [comment]
Reacendendo a discussão pq saiu uma [notícia interessante](https://tecnoblog.net/224255/comissao-europeia-estudo-pirataria/) sobre esse assunto.
